Develop an SQL schema for SOPs

```diff
@@ -0,0 +1,4351 @@
+
+
+--
+-- TOC entry 2932 (class 2606 OID 18164)
+-- Name: modelPathways modelPathways_id_key; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."modelPathways"
+    ADD CONSTRAINT "modelPathways_id_key" UNIQUE (id);
+
+
+--
+-- TOC entry 2934 (class 2606 OID 18166)
+-- Name: modelPathways modelPathways_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."modelPathways"
+    ADD CONSTRAINT "modelPathways_pkey" PRIMARY KEY (model_id, pathway_id);
+
+
+--
+-- TOC entry 2936 (class 2606 OID 18153)
+-- Name: modelReactions modelReactions_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."modelReactions"
+    ADD CONSTRAINT "modelReactions_pkey" PRIMARY KEY (id);
+
+
+--
+-- TOC entry 2996 (class 2606 OID 18084)
+-- Name: models models_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.models
+    ADD CONSTRAINT models_pkey PRIMARY KEY ("id ");
+
+
+--
+-- TOC entry 3000 (class 2606 OID 18155)
+-- Name: simulationParameters simulationParameters_ simulation_id_simulation_dateAndTime_key; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."simulationParameters"
+    ADD CONSTRAINT "simulationParameters_ simulation_id_simulation_dateAndTime_key" UNIQUE (" simulation_id", "simulation_dateAndTime");
+
+
+--
+-- TOC entry 3002 (class 2606 OID 18017)
+-- Name: simulationParameters simulationParameters_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."simulationParameters"
+    ADD CONSTRAINT "simulationParameters_pkey" PRIMARY KEY ("id ");
+
+
+--
+-- TOC entry 2998 (class 2606 OID 18095)
+-- Name: simulation simulation_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.simulation
+    ADD CONSTRAINT simulation_pkey PRIMARY KEY (id);
+
+
+--
+-- TOC entry 2914 (class 2606 OID 18157)
+-- Name: tracers tracers_met_id_met_name_key; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.tracers
+    ADD CONSTRAINT tracers_met_id_met_name_key UNIQUE (met_id, met_name);
+
+
+--
+-- TOC entry 2916 (class 2606 OID 18159)
+-- Name: tracers tracers_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.tracers
+    ADD CONSTRAINT tracers_pkey PRIMARY KEY (id);
+
+
+-- Completed on 2020-10-30 17:12:35
+
+--
+-- PostgreSQL database dump complete
+--
+
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 12.3
+-- Dumped by pg_dump version 12.3
+
+-- Started on 2020-10-30 17:14:47
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+DROP DATABASE "SOPs";
```

Comment:
As above, Let's change the name of the database so that they are all the same.

Reply:
done